user/test.py

```diff
@@ -2,32 +2,15 @@
 from unittest import TestCase
 
 from core import app
+from core.test import BaseTest
```

Comment:
you don't need to import this, it's an abstract class that you only need when extending.